src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -289,7 +294,7 @@ export function updateLayeredMaterialNodeElevation(context, layer, node, force)
                 terrain.texture.flipY = false;
                 terrain.texture.needsUpdate = true;
             }
-            node.setTextureElevation(terrain);
+            if (terrain.texture.image.data[0] && terrain.texture.image.data[terrain.texture.image.data.length - 1] > -99999) node.setTextureElevation(terrain);
```

Comment:
The test is not complete, I wanted to test diagonal extremity pixels. More like:
```
let terrainData = terrain.texture.image.data;
if (terrainData[0] > -99999 && terrainData[terrainData.length - 1] > -99999) 
        node.setTextureElevation(terrain);
```

It's a quick fix to avoid using non-real elevation tiles value that create holes.
As you asked @autra the IGN_MNT_HIGHRES DTM is complementary with the WORLD_DTM now that replaces the low res IGN_MNT (We should rename all MNT with DTM also).
So when declaring them, their TileMatrixSet have an empty union. I should try to see how it works when we don't take care of that.

@peppsac Yes I think it should be done as we do for color layers, with an order sequence for elevation layers. To avoid overloading the number of textures, the active elevation layer choice should be done before sending elevation to the shader.